From 7ad6ff1b3e3d6d078ad7fb223faa72b36d311d37 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Thu, 15 Sep 2016 15:01:48 -0600
Subject: [PATCH 09/20] uhd: Fix order of include dirs

Boost_INCLUDE_DIRS could previously "undo" UHD_INCLUDE_DIRS if it
pointed to the same location as a UHD installation that was *not*
selected via UHD_INCLUDE_DIRS.
---
 gr-uhd/lib/CMakeLists.txt  | 2 +-
 gr-uhd/swig/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gr-uhd/lib/CMakeLists.txt b/gr-uhd/lib/CMakeLists.txt
index b57b80b..b30a0d6 100644
--- a/gr-uhd/lib/CMakeLists.txt
+++ b/gr-uhd/lib/CMakeLists.txt
@@ -21,12 +21,12 @@
 # Setup the include and linker paths
 ########################################################################
 include_directories(
+    ${UHD_INCLUDE_DIRS}
     ${CMAKE_CURRENT_BINARY_DIR}
     ${GR_UHD_INCLUDE_DIRS}
     ${GNURADIO_RUNTIME_INCLUDE_DIRS}
     ${LOG4CXX_INCLUDE_DIRS}
     ${Boost_INCLUDE_DIRS}
-    ${UHD_INCLUDE_DIRS}
 )
 
 link_directories(
diff --git a/gr-uhd/swig/CMakeLists.txt b/gr-uhd/swig/CMakeLists.txt
index ef54761..163ca8f 100644
--- a/gr-uhd/swig/CMakeLists.txt
+++ b/gr-uhd/swig/CMakeLists.txt
@@ -26,10 +26,10 @@ include(GrSwig)
 set(GR_SWIG_FLAGS -DGR_HAVE_UHD) #needed to parse uhd_swig.i
 
 set(GR_SWIG_INCLUDE_DIRS
+    ${UHD_INCLUDE_DIRS}
     ${GR_UHD_INCLUDE_DIRS}
     ${GNURADIO_RUNTIME_SWIG_INCLUDE_DIRS}
     ${Boost_INCLUDE_DIRS}
-    ${UHD_INCLUDE_DIRS}
 )
 
 if(ENABLE_GR_CTRLPORT)
-- 
2.1.4

